### PR TITLE
example for cloud_only deployment with azurehpc

### PR DESCRIPTION
**Pull request by hmeiland**

### CODE REVIEW

File: tutorials/onboarding/cloud_only.json [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+      "script": "disable-selinux.sh",
+      "tag": "disable-selinux",
+      "sudo": true
+    },
+    {
+      "script": "cndefault.sh",
+      "tag": "cndefault",
+      "sudo": true
+    },
+    {
```
Comment (kanchanm) [inline tutorials/onboarding/cloud_only.json:113]: replace with ANF

**Comment by kanchanm** [comment]
add readme

File: tutorials/onboarding/cloud_only.json (hunk 1 above)
Comment (hmeiland) [reply tutorials/onboarding/cloud_only.json:113]: done

**Comment by hmeiland** [comment]
> 
> 
> add readme

done

File: examples/onboarding/readme.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+example for onboarding
+
+to use:
+```
+#> git clone https://github.com/Azure/azurehpc
+#> source azurehpc/install.sh
+#> azhpc-init -c $azhpc_dir/examples/onboarding -d onboarding -v resource_group=azurehpc-cluster,location=westeurope, win_password=WindowsPassword@12345!
```
Comment (xpillons) [inline examples/onboarding/readme.md:8]: Please don't put passwords even if wrong. Instead use something like [PutYourPassword]

**Comment by hmeiland** [comment]
Passwd done; not sure how to move changes to other branch

Hugo

File: tutorials/onboarding/cloud_only.json [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+      "tag": "loginnode",
+      "sudo": false
+    },
+    {
+      "script": "pbsserver.sh",
+      "copy": [
+        "pbspro_19.1.1.centos7/pbspro-server-19.1.1-0.x86_64.rpm"
+      ],
+      "tag": "pbsserver",
+      "sudo": false
```
Comment (xpillons) [inline tutorials/onboarding/cloud_only.json:145]: please change to true for all pbs scripts

File: examples/onboarding/config.json [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+      "tag": "loginnode",
+      "sudo": false
+    },
+    {
+      "script": "pbsserver.sh",
+      "copy": [
+        "pbspro_19.1.1.centos7/pbspro-server-19.1.1-0.x86_64.rpm"
+      ],
+      "tag": "pbsserver",
+      "sudo": false
```
Comment (xpillons) [inline examples/onboarding/config.json:177]: please change sudo to true for all pbs install scripts

Comment (hmeiland) [reply examples/onboarding/config.json]: done

File: tutorials/onboarding/cloud_only.json (hunk 3 above)
Comment (hmeiland) [reply tutorials/onboarding/cloud_only.json]: done

File: examples/onboarding/readme.md (hunk 2 above)
Comment (hmeiland) [reply examples/onboarding/readme.md]: done

**Pull request merged**